src/private_key_data.ts

```diff
@@ -31,6 +31,14 @@ export class PrivateKeyData {
 	private unverifiedDriveKeys: DriveKey[];
 
 	constructor({ password, driveKeys, wallet }: PrivateKeyDataParams) {
+		if (password && !wallet) {
+			throw new Error('Password supplied without a wallet. Did you forget to include your wallet?');
+		}
+
+		if (password && driveKeys) {
+			throw new Error("Password and drive keys can't be used together. Please provide one or the other.");
```

Comment:
This string is wrapped in double quotes here, I'm surprised prettier format and save and the husky linter hook let that through.

Reply:
Also, I don't think this wrapper in particular would care if both `password && driveKeys` were supplied. It will always try the supplied drive keys first, then fallback to using the password if no driveKeys can decrypt.